index.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Saarc Summit</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header>
+        <nav class="top-navbar">
+            <ul class="top-nav-items">
+                <li><a href="#"><img src="./assets/icons/facebook.png" alt="facebook"></a></li>
+                <li><a href="#"><img src="./assets/icons/twitter.png" alt="twitter"></a></li>
+                <li><a href="#" class="link-top-nav">English</a></li>
+                <li><a href="#" class="link-top-nav">My Page</a></li>
+                <li><a href="#" class="link-top-nav">Logout</a></li>
+            </ul>
+        </nav>
+        <nav class="navbar">
+            <div class="nav-img-container">
+            <a href="#" class="navbar-img-logo"><img src="./assets/images/logo.png" alt="Saarc Logo"></a>
+            </div>
+            <div class="hamburger">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
+
+            <ul class="nav-menu">
+                <li class="nav-item">
+                    <a href="./aboutme.html" class="nav-link">About</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#" class="nav-link">Program</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#" class="nav-link">Join</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#" class="nav-link">Sponser</a>
+                </li>
+                <li class="nav-item">
+                    <a href="#" class="nav-button">Campaign</a>
+                </li>
+            </ul>
+            
+        </nav>
+    </header>
+    <main>
+        <!--First Section-->
+        <section class="first-section">
+            <div>
+            <p class="text-1">Dear Viewer's Welcome to the</p>
+            <h1 class="main-text">The 20th SAARC Summit</h1>
+            </div>
```

Comment:
- [ ] Please use the exact same colors for text from the original design. Kindly change every occurence of `#ae8625` in this section to `#ec5242`. This is a project requirement 👍 
### Your Design
![image](https://user-images.githubusercontent.com/46839548/176976305-86636a85-6672-41af-873b-5491f14756e0.png)
### Behance Design
![image](https://user-images.githubusercontent.com/46839548/176976327-9bb5d51b-3c6a-4921-8d89-ba3915091e09.png)